src/applications/vaos/utils/constants.js

```diff
@@ -3,6 +3,7 @@ export const FETCH_STATUS = {
   notStarted: 'notStarted',
   succeeded: 'succeeded',
   failed: 'failed',
+  failedVaos400: 'failedVaos400',
```

Comment:
I had to make the choice between adding a new FETCH_STATUS or adding another field such as "isBadRequest" to the redux state and decided to go with the former.

Reply:
I think I'd prefer adding a new field, it's still a failed state. Now we have to make sure to catch two different failure states whenever we use this status constant, which I think is going to be easy to forget.